images/linux/scripts/installers/docker.sh

```diff
@@ -26,9 +26,4 @@ docker pull debian:9
 docker pull alpine:3.7
 docker pull alpine:3.8
 docker pull alpine:3.9
-docker pull alpine:3.10
-
-## Add version information to the metadata file
-echo "Documenting Docker version"
-DOCKER_VERSION=`docker -v`
```

Comment:
Just wonder, why we delete these lines.
Previously, `Docker` doc entry was produced in 2 places:
1) docker-moby (that you fixed)
2) docker (that you have removed)
So where the original `Docker` is installed? Or we don't install original docker at all?

Reply:
Its shows same thing.
https://github.com/actions/virtual-environments/releases/tag/ubuntu18%2F20200217.1